### PR TITLE
make changes to sticky-ad 1.0, fix tests

### PR DESCRIPTION
**Pull request by zhouyx**
@erwinmombay I fixed the tests that we discussed offline today by specifically declaring version number. PTAL

### CODE REVIEW

File: extensions/amp-sticky-ad/1.0/amp-sticky-ad.css [hunk 1]
```diff
@@ -23,6 +23,10 @@ amp-sticky-ad {
   z-index: 11;
   max-height: 100px !important;
   box-sizing: border-box;
+  opacity: 1 !important;
+  background-image: none !important;
+  background-color: #fff;
+  padding-top: 2px;
```
Comment (dvoytenko) [inline extensions/amp-sticky-ad/1.0/amp-sticky-ad.css]: Why padding-top: 2px?

File: extensions/amp-sticky-ad/0.1/amp-sticky-ad.js [hunk 2]
```diff
@@ -238,6 +238,6 @@ class AmpStickyAd extends AMP.BaseElement {
   }
 }
 
-AMP.extension('amp-sticky-ad', AMP => {
+AMP.extension('amp-sticky-ad:0.1', AMP => {
```
Comment (dvoytenko) [inline extensions/amp-sticky-ad/0.1/amp-sticky-ad.js]: I'm sort of surprised this didn't crash many other tests....

File: extensions/amp-sticky-ad/1.0/amp-sticky-ad.css (hunk 1 above)
Comment (zhouyx) [inline extensions/amp-sticky-ad/1.0/amp-sticky-ad.css]: Oh, I was experimenting with this PR with the new button style. Failed to clean up. Removed

File: extensions/amp-sticky-ad/0.1/amp-sticky-ad.js (hunk 2 above)
Comment (zhouyx) [inline extensions/amp-sticky-ad/0.1/amp-sticky-ad.js]: Yup, but to all other tests i think `amp-sticky-ad` and `amp-sticky-ad:0.1` makes no difference to them, it's just a symbol name that represent the id the of amp-sticky-ad installer and can be named to whatever thing.

Comment (zhouyx) [inline extensions/amp-sticky-ad/0.1/amp-sticky-ad.js]: Actually `amp-sticky-ad` is only used in its own tests and the `amp-ad-3p-impl` tests. For the ad test, we don't really care about stickyAd class, all we want is a container with name `amp-sticky-ad`

**Pull request merged**